packages/task/src/browser/task-service.ts

```diff
@@ -166,7 +166,7 @@ export class TaskService implements TaskConfigurationClient {
             if (task) {
                 taskIdentifier = this.taskNameResolver.resolve(task);
             }
-            this.messageService.info(`Task ${taskIdentifier} has been started`);
+            this.messageService.info(`Task \'${taskIdentifier}\' has been started`);
```

Comment:
[minor] add punctuation to align with other messages?

https://github.com/eclipse-theia/theia/pull/6668/files#diff-794cdf6ef93aef7cc827f31f92ebe226R220
https://github.com/eclipse-theia/theia/pull/6668/files#diff-794cdf6ef93aef7cc827f31f92ebe226R227